tool/src/org/antlr/v4/codegen/UnicodeEscapes.java

```diff
@@ -11,12 +11,6 @@
  * languages' syntax.
  */
 public class UnicodeEscapes {
-	public static String escapeCodePoint(int codePoint, String language) {
```

Comment:
so it's not used anywhere but TestUnicodeEscapes?

Reply:
Yes.